recipes/pytest-update-test-results/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "pytest-update-test-results" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ESSS/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c5f2a3400a2b65d63df8e789d77de2a5338b15ce56c5afa5518a908cac56d0dd
+
+build:
+  number: 0
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
```

I think it still needs `setuptools_scm` as a host requirement, otherwise the internal version info is still 0.0.0.

```suggestion
    - setuptools_scm
    - pip
```